### DISCUSSION TITLE
pro-layout组件能否增加一个弹出式导航

### DISCUSSION THREAD

**Discussion by yunuo-tech** [ant-design/pro-components#273] [Ideas]
就是设计规范这里讲到的
![image](https://user-images.githubusercontent.com/5960864/90948264-772a8700-e46f-11ea-87ac-ab893e577a05.png)

目前菜单层级多了后确实会在操作上有点不方便
![image](https://user-images.githubusercontent.com/5960864/90948283-a04b1780-e46f-11ea-9b6a-31c14ce098c8.png)

**Comment by zhangzhikun** [comment]
兄弟找到解决方案了吗？
